helm/values.yaml

```diff
@@ -0,0 +1,92 @@
+bootstrap:
+  image:
+    #must be postgres or mysql
+    repository: postgres
+    pullPolicy: IfNotPresent
+    tag: 16.1-bullseye
```

Comment:
Here we do not really specify postgres version, so I think we should make this same; let user to define, use latest, or fix version. 
- https://github.com/trinodb/trino-gateway/blob/main/docker/postgres-backend-compose.yml#L4

Reply:
sure